toml/src/main/resources/META-INF/toml-only.xml

```diff
@@ -27,10 +27,20 @@
                          displayName="Cyclic feature dependency"
                          enabledByDefault="true" level="ERROR"
                          implementationClass="org.rust.toml.inspections.CargoTomlCyclicFeatureInspection"/>
-        <localInspection language="TOML" groupName="Rust"
+        <localInspection language="TOML"
                          displayName="Crate not found"
-                         enabledByDefault="true" level="WARNING"
+                         groupPath="Rust,Cargo.toml"
+                         groupName="Dependencies"
```

Comment:
Looks like https://github.com/intellij-rust/intellij-rust/pull/6801#discussion_r611579058 is still relevant